recipes-domu/domu-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/defconfig

```diff
@@ -4587,7 +4591,23 @@ CONFIG_ARCH_TEGRA_210_SOC=y
 # CONFIG_SOC_TI is not set
 CONFIG_RCAR_THERMAL_EMS_ENABLED=y
 CONFIG_RCAR_DDR_BACKUP=y
-# CONFIG_PM_DEVFREQ is not set
+CONFIG_PM_DEVFREQ=y
```

Comment:
What have changed in the system so that you have CONFIG_PM_DEVFREQ enabled?

Reply:
See above.

Reply:
As for you exact question, I think that CONFIG_PM_DEVFREQ was enabled in order to fit new DDK which did support GSX frequency scaling.